tests/admin-mode_test.go

```diff
@@ -0,0 +1,194 @@
+	scheme "k8s.io/client-go/kubernetes/scheme"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var (
+	kubeconfig     string
+	config         *restclient.Config
+	client         *kubernetes.Clientset
+	clientSet      *chaosClient.LitmuschaosV1alpha1Client
+	err            error
+	experimentName = "pod-delete"
+	engineName     = "adminengine"
+)
+
+func TestChaos(t *testing.T) {
+
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "BDD test")
+}
+
+var _ = BeforeSuite(func() {
+	var err error
+	kubeconfig = os.Getenv("HOME") + "/.kube/config"
+	config, err = clientcmd.BuildConfigFromFlags("", kubeconfig)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get config")
+	}
+
+	client, err = kubernetes.NewForConfig(config)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get client")
+	}
+
+	clientSet, err = chaosClient.NewForConfig(config)
+
+	if err != nil {
+		Expect(err).To(BeNil(), "failed to get clientSet")
+	}
+
+	err = v1alpha1.AddToScheme(scheme.Scheme)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	//Checking the status of operator which is already installed and running
+	operator, err := client.AppsV1().Deployments(chaosTypes.ChaosNamespace).Get("litmus", metav1.GetOptions{})
+	Expect(err).To(BeNil(), "Failed to get operator")
+	count := 0
+	for operator.Status.UnavailableReplicas != 0 {
+		if count < 50 {
+			fmt.Printf("Unavaliable Count: %v \n", operator.Status.UnavailableReplicas)
+			operator, err = client.AppsV1().Deployments(chaosTypes.ChaosNamespace).Get("litmus", metav1.GetOptions{})
+			Expect(err).To(BeNil(), "Failed to get operator")
+			time.Sleep(5 * time.Second)
+			count++
+		} else {
+			Fail("Operator is not in Ready state Time Out")
+		}
+	}
+	fmt.Println("Chaos Operator is up and Running")
+})
+
+//BDD Tests for testing admin mode functionality
+//Checking for the creation of runner pod for application in same namespace
+var _ = Describe("BDD of operator reconcile resiliency check", func() {
+
+	// BDD TEST CASE 1
+	Context("Check for litmus components", func() {
+
+		It("Should check for creation of runner pod", func() {
+
+			//Create Namespace for the test
+			By("Creating namespace for the test")
+			cmd := exec.Command("kubectl", "create", "ns", "test")
+			cmd.Stderr = os.Stderr
+			err := cmd.Run()
+			if err != nil {
+				log.Fatalf("cmd.Run() failed with %s\n", err)
+			}
+
+			//Creating application for pod-delete in test namespace
+			By("Creating deployment for pod-delete chaos")
+			err = exec.Command("kubectl", "run", "adminapp", "--image=nginx").Run()
+			Expect(err).To(BeNil(), "Failed to create deployment")
+			fmt.Println("Test Application is created")
+
+			//Installing RBAC for experiment that is pod-delete
+			rbacPath := "https://raw.githubusercontent.com/litmuschaos/pages/master/docs/litmus-admin-rbac.yaml"
+			err = exec.Command("wget", "-O", experimentName+"-sa.yaml", rbacPath).Run()
```

Comment:
You can rename this rbac to admin-rbac.yaml

Reply:
We already get this file in BDD1 so can you remove this step here?

Reply:
same as above